src/frontend/middleware.go

```diff
@@ -58,6 +58,8 @@ func (lh *logHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	requestID, _ := uuid.NewRandom()
 	ctx = context.WithValue(ctx, ctxKeyRequestID{}, requestID.String())
 
+	http_request_count.Add(ctx, 1)
```

Comment:
There should be a built-in metric for instrumenting grpc with opentelemetry that should capture this. If this doesn't exist, then it's fine to leave it like this for now. Perhaps worth adding a //todo comment to remove it in the future when such metric is implemented in the core OT library.

Reply:
There isn't a Views API and this isn't collected automatically, so it's custom metrics for now. With `othttp` it has some automatic ones but not the exact ones we want.